api/v1/kubtest.yaml

```diff
@@ -409,6 +422,37 @@ components:
         execution:
           $ref: '#/components/schemas/Execution'
 
+    ExecutionsResult:
+      description: the result for a page of executions
+      type: object
+      properties:
+        totals:
+          $ref: '#/components/schemas/ExecutionTotals'
+        results:
+          type: array
+          items:
+            $ref: '#/components/schemas/ExecutionSummary'
+
+    ExecutionTotals:
+      type: object
+      description: various execution counters
+      properties:
+        results:
+          type: integer
+          description: the total number of executions available
+        passed:
```

Comment:
Thought it worth mentioning it here maybe to not forget, it would be great if we make it consistent with `ExecutionStatus` enum below, other than that LGTM! 😃